modules/config.js

```diff
@@ -19,5 +20,24 @@ module.exports = {
 
   setMatchMedia (nextMatchMediaFunction: Function) {
     _matchMediaFunction = nextMatchMediaFunction;
+  },
+
+  setMediaQueryAlias (alias: string, query: string) {
+    if (alias.indexOf('@') !== 0) {
```

Comment:
Yes, and it [seems pretty easy](https://github.com/petehunt/webpack-howto#6-feature-flags) to add with webpack.